src/zenml/orchestrators/utils.py

```diff
@@ -0,0 +1,62 @@
+import time
+from typing import Optional
+
+from tfx.orchestration.portable import data_types, launcher
+
+from zenml.exceptions import DuplicateRunNameError
+from zenml.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+def _format_timedelta_pretty(seconds: float) -> str:
```

Comment:
I think we can put this somewhere that is more accessible?

Reply:
I think `cli/utis.py` has a similar method. maybe merge these and put in general utils ?

Reply:
zenml/utils/logging_utils? zenml/utils/formatting_utils? zenml/utils/string_utils? Any other suggestion?

Reply:
` zenml/utils/string_utils`  sounds good